src/styles.scss

```diff
@@ -1,12 +1,42 @@
-$color-background-main: #fafafa;
+$color-background: #fafafa;
+$color-primary: #37ac00;
+$color-primary-select: #65d132;
+$color-grey: #7a7a7a;
+$color-grey-light: #e5e5e5;
```

Comment:
It is very fine idea to create variables for colors, but I think it would be better to make this separate file(for example: colors.scss)